src/app/options.cpp

```diff
@@ -247,6 +252,12 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"path"};
+    constexpr const StringOption STATUS_OPTION = {"status"};
```

Comment:
I spent some time thinking about what the name of this option should be, and settled on `status` (now `torrent-status`) because I can't find any other terminology used elsewhere in the app that refers to the attribute of a torrent that distinguishes whether it's paused or not in the way that we need.

In the add torrent dialog, there is a check box that reads "Start torrent", but since we're overriding the default behavior with this command line option, a simple toggle isn't good enough. The user needs to specify one of three states: use default settings, pause torrent, start torrent. The `torrent-status` approach allows for all three.

The actual setting in the preferences for specifying the default behavior is "Do not start the download automatically" which doesn't really help for coming up with an intuitive command line option either.

In the main window, whether a torrent is paused or running is indicated in the status column.

So I eventually settled with `torrent-status`. I originally set it up so that there was an `--add-paused` and `--add-started` option but a while back @glassez suggested that was redundant and I'm inclined to agree. I think `torrent-status=paused|started` is the way to go.